spec/controllers/admin/processors_controller_spec.rb

```diff
@@ -0,0 +1,95 @@
+# frozen_string_literal: true
+
+describe Admin::ProcessorsController, type: :controller do
+  describe '#create' do
+    subject { post :create }
+
+    let(:hodinkee_file) { file_fixture('shophodinkee_first_page.html').read }
+    let(:crown_file) { file_fixture('crownandcaliber_first_page.html').read }
+    let(:bobs_file) { file_fixture('bobswatches_first_page.html').read }
+
+    before do
+      allow_any_instance_of(Browser).to receive(:visit)
+
+      allow_any_instance_of(Browser)
+        .to receive(:visit)
+        .with(url: 'https://shop.hodinkee.com/collections/watches?page=18',
```

Comment:
`https://shop.hodinkee.com/collections/watche.......` should be used as variable